mix.exs

```diff
@@ -7,7 +7,6 @@ defmodule AppPrototype.Mixfile do
      elixir: "~> 1.5.1",
      elixirc_paths: elixirc_paths(Mix.env),
      compilers: [:phoenix, :gettext] ++ Mix.compilers,
-     build_embedded: Mix.env == :prod,
      start_permanent: Mix.env == :prod,
```

Comment:
@craiglyons This might be an issue in a different PR, but if our convention is now `MIX_ENV=production`, then should this be `Mix.env == :production`?

Reply:
For posterity, we agreed to normalize on `prod`. 👍